### ISSUE TITLE
Constructing new mels as Input

### ISSUE THREAD

**Issue by ZohaibAhmed** [fatchord/WaveRNN#19]
@fatchord, thanks for your work on this. The samples you have are fantastic, and the model converges really quickly.

How do I go about creating the mel as the input? Do I need to train another model that produces mels and pipe that as the input? Or should I be able to take any wav file, construct a mel, and pass that as the input?

**Comment by G-Wang** [comment]
In the preprocessing ipython notebook, you can see the steps used to get Mel spectrlgram from audio wav.

WavRNN is by itself a vocoder, so it only takes in Mel spectrogram. You can pass the Mel spectrogram generated by a text to Mel model, such as tacotron. 

You need to make sure that the Mel spectrogram processing is the same for both parts.

**Comment by bjtommychen** [comment]
Hi, all
I don't understand what the input is.
for 1/2/3, it's wave data clearly.
but for 4b. it's wave data + melspectrogram.  how can feed this two into WaveRNN() of 1/2/3.
Thanks.

**Comment by fatchord** [comment]
@ZohaibAhmed Yeah, that tacotron2 repo is absolutely fine for creating training mels. I used my own implementation of tacotron for the demo samples.

**Comment by fatchord** [comment]
@bjtommychen You're correct, notebooks 1, 2 & 3 are unconditioned. However if shouldn't be too difficult to add the upsampling network from 4b to the original WaveRNN model if you feel so inclined.
